js/packages/openinference-instrumentation-langchain/jest.config.js

```diff
@@ -3,4 +3,5 @@ module.exports = {
   preset: "ts-jest",
   testEnvironment: "node",
   prettierPath: null,
+  setupFiles: ["dotenv/config"],
```

Comment:
This needs to be removed overall since it won't be a unit test

Reply:
Was going to remove before the release but not sure i follow the "since it won't be a unit test"

Reply:
it's not a unit test if it requires a .env file. I'd probably just set your keys via your shell right now so you don't need this but NBD

Reply:
ahh i see what you mean, my keys are in my shell but for some reasonweren't getting picked up without this in iether case will definitely remove before final merge